README.md

```diff
@@ -1 +1,181 @@
-# Hello-Microverse-Project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="codeeq.jpg" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Hellow Microverse Project</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 Hello Microverse Project <a name="about-project"></a>
+
+This is my First Microverse Project. I created it using HTML and CSS. In addition, i added Linters for Html and Css.
+
+## 🛠 Built With <a name="built-with"></a>
+
+1-HTML
+2-CSS
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Add index.html file.
+- Add style.css file.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<a href="https://codeeq-verse.netlify.app/">Live Demo Link</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- Hey, your live demo link is currently broken. please, kindly update it to be working or kindly remove this live demo section from the readme body and from the top navigation on the readme to make it more professional.

Reply:
This is fixed, kindly check again.
Thank you.